Sources/Brave/Frontend/Browser/TabManager.swift

```diff
@@ -916,20 +931,19 @@ class TabManager: NSObject {
                        flushToDisk: false,
                        zombie: true,
                        id: savedTab.tabId,
-                       isPrivate: false)
-      
-      let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
+                       isPrivate: savedTab.isPrivate)
       
       tab.lastTitle = savedTab.title
       tab.favicon = FaviconFetcher.getIconFromCache(for: tabURL) ?? Favicon.default
       tab.setScreenshot(savedTab.screenshot)
       
       Task { @MainActor in
-        tab.favicon = try await FaviconFetcher.loadIcon(url: tabURL, kind: .smallIcon, persistent: !isPrivateBrowsing)
+        tab.favicon = try await FaviconFetcher.loadIcon(url: tabURL, kind: .smallIcon, persistent: tab.isPrivate)
```

Comment:
It should be reverse, no? Now it persists in private mode, forgot the reverse exclamation mark